docs/how-tos/manage-fleet.md

```diff
@@ -130,11 +130,24 @@ You can manage this in the **API Keys** section of the organization settings pag
 
 ## Monitoring and debugging
 
-Viam allows you to access each machine, check its logs, check recent changes, and roll back changes if needed.
+Viam allows you to view all machines from a dashboard and access each machine, check its logs, check recent changes, and roll back changes if needed.
 
 {{< table >}}
+{{% tablestep %}}
+**1. Monitor your fleet's synced data and part status**
+
+You can monitor the amount of binary and tabular data your fleet has synced in the last 48 hours from the Viam app's **FLEET** tab's [**DASHBOARD** subtab](https://app.viam.com/fleet/dashboard).
+You can also monitor the number of parts online, offline, and awaiting setup on this dashboard.
```

Comment:
```suggestion
You can monitor the number of parts online, offline, and awaiting setup on this dashboard.
You can also monitor the amount of binary and tabular data your fleet has synced in the last 48 hours from the Viam app's **FLEET** tab's [**DASHBOARD** subtab](https://app.viam.com/fleet/dashboard).
```

Reply:
sorry, why this change? we don't explain what "this dashboard" is until the second sentence so it makes sense for it to come first

Reply:
@npentrel just want to check and make sure you're ok with the original phrasing before I merge this

Reply:
The same reason I flipped the title for this section. That's not the order the information appears on the page. 

Is there a reason why you're putting the binary and tabular data amounts first? 

```suggestion
You can monitor the number of parts online, offline, and awaiting setup from the Viam app's **FLEET** tab's [**DASHBOARD** subtab](https://app.viam.com/fleet/dashboard).
You can also monitor the amount of binary and tabular data your fleet has synced in the last 48 hours on this dashboard.
```

Reply:
Ahh yes that makes sense. I was just thinking that the data views are the main feature of the page so it made sense to say so first but happy with this suggestion. thanks!